SQLQuery_Week09_Homework.sql

```diff
@@ -0,0 +1,414 @@
+CREATE DATABASE LibrariesManagement;
+
+USE LibrariesManagement;
+
+CREATE TABLE Libraries
+(
+    Id INT IDENTITY(1, 1) PRIMARY KEY,
+    Name VARCHAR(50),
+    StreetName VARCHAR(50),
+    City VARCHAR(50)
+)
+
+CREATE TABLE Librarians
+(
+    Id INT IDENTITY(1, 1) PRIMARY KEY,
+    Name VARCHAR(30),
+    HireDate Date,
+    IsOnHoliday VARCHAR(3),
+    LibraryId INT REFERENCES Libraries(Id)
+)
+
+CREATE TABLE Books
+(
+    Id INT IDENTITY(1, 1) PRIMARY KEY,
+    Title VARCHAR(50),
+    AuthorFirstName VARCHAR(30),
+    AuthorLastName VARCHAR(30),
+    Type VARCHAR(10), -- crime, fantasy, horror, adventure
+)
+
+CREATE TABLE Books_Libraries
+(
+    Id INT IDENTITY(1, 1) PRIMARY KEY,
+    BookId INT REFERENCES Books(Id),
+    LibraryId INT REFERENCES Libraries(Id)
+)
+
+CREATE TABLE Sales
+(
+    Id INT IDENTITY(1, 1) PRIMARY KEY,
+    NumberOfCopies INT,
+    LibraryId INT REFERENCES Libraries(Id),
+    LibrarianId INT REFERENCES Librarians(Id)
+)
+
+CREATE TABLE Books_Sales
+(
+    Id INT IDENTITY(1, 1) PRIMARY KEY,
+    BookId INT REFERENCES Books(Id),
+    SalesId INT REFERENCES Sales(Id)
+)
+
+-- column LibraryID is redundanted. We can join with Sales table for connect with Library table
```

Comment:
Love the comments. They make my life easier when reviewing your homework and it also makes it clear that you know how to explain what you are doing.